doc/howto/import_machines_to_instances.md

```diff
@@ -69,11 +69,6 @@ Complete the following steps to migrate an existing machine to a LXD instance:
 
    The tool then asks you to provide the information required for the migration.
 
-   ```{tip}
```

Comment:
should be a separate commit - not related to commit message. Needs to be backported.